site/src/js/widgets/PeopleDirectoryWidget/views/NCIOrgDirectoryView.html

```diff
@@ -535,5 +535,6 @@ <h3 class="h4">{{FilterHeader}}</h3>
         });
         RenderChildWidgets('#organizationdirectorytabwidget');        
     });
-    window.LoadOrganizationDirectoryTreeFilter();
+    window.peopleDirectory_html_loadOrgs();
+    window.peopleDirectory_html_loadTabs();
```

Comment:
I think, and this is very much a guess, that based on the order previously processed we should flip these two.